### ISSUE TITLE
Questions about visualization in the paper

### ISSUE THREAD

**Issue by zzw1123** [liutinglt/CE2P#17]
Thanks for your contribution!
I have noticed that in the paper, some visualization results are provided in Fig.4. 
However, there seems like a vertical line divide the small blue person(surrounded by white bounding box) into two parts. 
And there is no texture difference between this two parts. I am wondering the reason.
![image](https://user-images.githubusercontent.com/16879220/50215123-10f98100-03bd-11e9-9691-a44cf0f92eb1.png)

**Comment by RanTaimu** [comment]
@zzw1123 When the individual parsing results fused into the final result, the shared area of two segmentation maps will inference each other. In the case you mentioned, the shared area of the blue person and the nearby person covers the pixels around the "vertical line". For each of these pixels, the confidence on background is larger than that on foreground, leading to the phenomenon you mentioned.

**Comment by RanTaimu** [comment]
@zzw1123 Yes you're right. you can refer to the file 'M-CE2P-test.py' in [M-CE2P](https://github.com/RanTaimu/M-CE2P) for more details.

**Comment by zzw1123** [comment]
@RanTaimu Got it! Thank you so much for the help!
